data/json/monstergroups.json

```diff
@@ -1219,7 +1221,24 @@
       { "monster" : "mon_zombie_brute", "freq" : 15, "cost_multiplier" : 15 },
       { "monster" : "mon_zombie_hulk", "freq" : 1, "cost_multiplier" : 50 },
       { "monster" : "mon_zombie_master", "freq" : 1, "cost_multiplier" : 30 },
-      { "monster" : "mon_zombie_hollow", "freq" : 2, "cost_multiplier" : 10 }
+      { "monster" : "mon_zombie_hollow", "freq" : 2, "cost_multiplier" : 10 },
+      { "monster" : "mon_zombie_predator", "freq" : 5, "cost_multiplier" : 10 }
+    ]
+  },{
+    "type":"monstergroup",
+    "name" : "GROUP_FERAL_HUNTER",
+    "default" : "mon_null",
+    "//" : "Masters pick from here when upgrading; no dogs, bionics, or profession-types",
```

Comment:
Does the master really pick from that group? I thought they pick from "GROUP_ZOMBIE_UPGRADE"? What bout the next group, does the master pick from all those groups?